ros2topic/ros2topic/verb/echo.py

```diff
@@ -181,10 +181,15 @@ def main(self, *, args):
 
         # Validate field selection
         self.field = args.field
-        if self.field is not None:
-            self.field = list(filter(None, self.field.split('.')))
-            if not self.field:
-                raise RuntimeError(f"Invalid field value '{args.field}'")
+        if args.field:
+            for field in args.field:
+                try:
+                    value = get_field_value(message, field)
```

Comment:
where is this message come from? originally this part is to validate the fields only, not printing the message. (there is no message received at here...)

Reply:
okay then i will remove this i thought i need to send the message

Reply:
and i think this needed a little bit changes which i will do so i don't think to convert into draft as per my opinion. What is your opinion in this?